File: SumOfNaturalNumber.c
#include <stdio.h>

int sumOfNaturalNumber(int n) {
    int sum = 0;
    for (int i = 1; i <= n; i++) { // Chạy từ 1 đến n
        sum += i;
    }
    return sum;
}

int main() {
    int n;
    printf("Hãy nhập một số n bất kỳ: ");
    scanf("%d", &n);

    if (n < 1) {
        printf("Hãy nhập số nguyên lớn hơn 0!\n");
    } else {
        // Gọi hàm và in kết quả
        int result = sumOfNaturalNumber(n);
        printf("Tổng các số tự nhiên từ 1 đến %d là: %d\n", n, result);
    }

    return 0;
}

File: SumOfNaturalNumber.py
def sumOfNaturalNumber(n):
    sum = 0
    for i in range(1,n+1):
        sum += i
    return sum

n = int(input("Hãy nhập một số tự nhiên bất kì: "))
if n < 1:
    print("Hãy nhập số tự nhiên lớn hơn 0!")
elif n == 1:
    print("1")
else:
    result = sumOfNaturalNumber(n)
    print(f"Kết quả tổng của các số từ 1 đến {n} là: {result}")
    